data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -204,6 +204,19 @@ podman run -d --net host --privileged --name image-customization \
     --secret pull-secret,mode=400 \
     ${CUSTOMIZATION_IMAGE}
 
+# We're not interesting for the exit code, just that the server is available
```

Comment:
It would actually be sufficient to just check if the container has exited after a short interval (perhaps after the sleep 10 on line 233).

Reply:
Are we sure 10 seconds are always enough? I guess it's not going to take too long...